.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+on:
+  push:
+    tags:
+    - '*'
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_VERSION: 1.76.0
```

Comment:
Here's one downside to multiple files - the rust version pin is not centralized.